### PR TITLE
Feature/rename library

### PR DESCRIPTION
**Pull request by benouaer**
### Change description ###
renaming files and folders to match new library name
**Does this PR introduce a breaking change?** (check one with "x")
```
[x] Yes
[ ] No
```

### CODE REVIEW

File: tsconfig.spec.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "./tsconfig.json",
```
Comment (linusnorton) [inline tsconfig.spec.json:2]: this makes no sense

File: tsconfig.app.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "./tsconfig.json",
```
Comment (linusnorton) [inline tsconfig.app.json]: this also makes no sense

File: tsconfig.spec.json (hunk 1 above)
Comment (benouaer) [reply tsconfig.spec.json:2]: true, let me check

File: tsconfig.app.json (hunk 2 above)
Comment (benouaer) [reply tsconfig.app.json]: this file needs to be deleted

File: tsconfig.spec.json (hunk 1 above)
Comment (benouaer) [reply tsconfig.spec.json:2]: actually, this is correct, why does it make no sense?

Comment (linusnorton) [reply tsconfig.spec.json:2]: it's extending itself

Comment (linusnorton) [reply tsconfig.spec.json:2]: ah sorry, no it's not

**Review by linusnorton — APPROVED** [review]

**Pull request merged**